ied-be/src/templates/certificates/ied/certificate.css

```
@@ -0,0 +1,256 @@
+  place-items: center;
+}
+
+.certificate-page {
+  position: relative;
+  width: var(--page-width);
+  height: var(--page-height);
+  padding: var(--page-padding);
+  overflow: hidden;
+  background:
+    linear-gradient(135deg, var(--paper) 0%, var(--paper-warm) 55%, rgba(201, 163, 78, 0.14) 100%);
+  page-break-after: always;
```

Stylelint (17.9.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)